extensions/arc/src/ui/dashboards/postgres/postgresPropertiesPage.ts

```diff
@@ -78,15 +77,15 @@ export class PostgresPropertiesPage extends DashboardPage {
 
 	private refresh() {
 		const endpoint: { ip?: string, port?: number } = this._postgresModel.endpoint;
-		const connectionString = `postgresql://postgres:${this._postgresModel.password}@${endpoint.ip}:${endpoint.port}`;
+		const connectionString = `postgresql://postgres@${endpoint.ip}:${endpoint.port}`;
 		const registration = this._controllerModel.getRegistration(ResourceType.postgresInstances, this._postgresModel.namespace, this._postgresModel.name);
 
 		this.keyValueContainer?.refresh([
 			new InputKeyValue(loc.coordinatorEndpoint, connectionString),
 			new InputKeyValue(loc.postgresAdminUsername, 'postgres'),
 			new TextKeyValue(loc.status, this._postgresModel.service?.status?.state ?? 'Unknown'),
 			new LinkKeyValue(loc.dataController, this._controllerModel.namespace ?? '', _ => vscode.window.showInformationMessage('TODO: Go to data controller')),
```

Comment:
@Charles-Gagnon  Is it possible to open the controller dashboard in a new tab here?  I figure something like vscode.commands.executeCommand('arc.openDashboard') but how do I pass parameters for which controller to open?

Reply:
With the changes I have in my PR we should be able to plumb the controller tree node through to the dashboard such that it could call openDashboard on that directly.

Another option (which is probably better TBH) is to modify the command to take in another identifier that it can use to find the correct treenode to open the dashboard for.

Reply:
Anyways I'd say get this in as it is and then we'll come back and hook up the dashboard links later (I have to do the same for MIAA so I'll probably just do them all in one go)